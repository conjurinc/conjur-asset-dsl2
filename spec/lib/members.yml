- !group &developers developers

- !members
  role: !user alice
  members:
    - *developers
    - 
      role: !group ops
      admin: true
