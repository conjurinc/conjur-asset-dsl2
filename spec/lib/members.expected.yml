---
- &1 !group
  id: developers
- !members
  role: !user
    id: alice
  members:
  - !member
    role: *1
  - !member
    role: !group
      id: ops
    admin: true
